rpms/antigen/antigen.spec

```diff
@@ -0,0 +1,29 @@
+Name: antigen
+Version: 2.2.3
+Release: 1%{?dist}
+Summary: A plugin manager for zsh, inspired by oh-my-zsh and vundle.
+
+License: MIT
+URL: https://github.com/zsh-users/antigen
+Source0: https://github.com/zsh-users/antigen/releases/download/v%{version}/v%{version}.tar.gz
+BuildRequires: make
+Requires: zsh
+Requires: git
+
+%description
+Antigen is a small set of functions that help you easily manage your shell (zsh) plugins, called bundles. The concept is pretty much the same as bundles in a typical vim+pathogen setup. Antigen is to zsh, what Vundle is to vim.
+
+%prep
+%setup -q
+
+%install
+make build
+make PREFIX=%{buildroot}/usr/share/zsh install
```

Comment:
- you are installing to `/usr/share/zsh/share/antigen.zsh`, the path is weird
 - use [macros](https://docs.fedoraproject.org/en-US/packaging-guidelines/RPMMacros/#_macros_for_paths_set_and_used_by_build_systems)
 - considering using `%make_install`, which sets `PREFIX` for you.

Reply:
if i used %make_install, it seems that PREFIX is not set.

```
+ /usr/bin/make install DESTDIR=/home/lxz/rpmbuild/BUILDROOT/antigen-2.2.3-1.fc34.x86_64 'INSTALL=/usr/bin/install -p'
mkdir -p /usr/local/share && cp /home/lxz/rpmbuild/BUILD/antigen-2.2.3/bin/antigen.zsh /usr/local/share/antigen.zsh
cp: cannot create regular file '/usr/local/share/antigen.zsh': Permission denied
```

I want the file to be installed in /usr/share/antigen.zsh, I need to set PREFIX to %{_prefix}, which is /usr, how should I write it?  thx :)

Reply:
my mistake, just use `make PREFIX=%{buildroot}%{_prefix}` is enough.